document/core/text/instructions.rst

```diff
@@ -214,6 +214,92 @@ Lexically, an |Toffset| or |Talign| phrase is considered a single :ref:`keyword
    \end{array}
 
 
+.. index:: atomic memory instruction
+   pair: text format; instruction
+.. _text-instr-atomic-memory:
+
+Atomic Memory Instructions
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. _text-atomic-load:
+.. _text-atomic-loadn:
+.. _text-atomic-store:
+.. _text-atomic-storen:
+.. _text-atomic-rmw:
+.. _text-atomic-rmwn:
+
+The offset immediate to atomic memory instructions is optional, and defaults to
+:math:`\T{0}`.
+
+.. math::
```

Comment:
Ah, I see. I was wondering why that was done elsewhere. I find the misalignment of the columns to be really ugly, and I assumed that the phantom stuff was meant to try to fix that, but it didn't seem to be working for me.